backend/src/src-vulnerabilities/src/main/java/org/eclipse/sw360/vulnerabilities/VulnerabilityHandler.java

```diff
@@ -128,6 +130,12 @@ public List<VulnerabilityDTO> getVulnerabilitiesByProjectIdWithoutIncorrect(Stri
         return getVulsByReleaseIdsWithoutIncorrect(releaseIds, user);
     }
 
+    @Override
+    public List<VulnerabilityDTO> getvulnerabilityDTOByExternalId(Set<String> externalIds, String releaseId) throws SW360Exception {
```

Comment:
Please use proper camel casing for method name (getvulnerabilityDTOByExternalId).
Before changing the name to getVulnerabilityDTOByExternalId check the service method in Sw360VulnerabilityService.java as it already have same method name in order to avoid confusion.